experiment/test_config.yaml

```diff
@@ -791,7 +795,7 @@ jobs:
     interval: 2h
     sigOwners: ['sig-gcp']
   ci-kubernetes-e2e-gce-cos-k8sstable3-reboot:
-    interval: 6h˙
+    interval: 6h
```

Comment:
😂 where's that coming from

Reply:
http://graphemica.com/%CB%99
Maybe someone bumped some keys on their mac? 🤔

Reply:
probably me